lib/Ed25519Signature2020.js

```diff
@@ -107,19 +110,27 @@ export class Ed25519Signature2020 extends LinkedDataSignature {
   }
 
   async assertVerificationMethod({verificationMethod}) {
+    let contextUrl;
+    let requiredKeyType;
+    if(verificationMethod.type === 'Ed25519VerificationKey2018') {
+      contextUrl = SUITE_CONTEXT_URL_2018;
+      requiredKeyType = 'Ed25519VerificationKey2018';
+    } else if(verificationMethod.type === 'Ed25519VerificationKey2020') {
+      contextUrl = SUITE_CONTEXT_URL;
+      requiredKeyType = 'Ed25519VerificationKey2020';
+    }
     if(!_includesContext({
-      document: verificationMethod, contextUrl: SUITE_CONTEXT_URL
+      document: verificationMethod, contextUrl
     })) {
       // For DID Documents, since keys do not have their own contexts,
       // the suite context is usually provided by the documentLoader logic
       throw new TypeError(
-        `The verification method (key) must contain "${SUITE_CONTEXT_URL}".`
+        `The verification method (key) must contain "${contextUrl}".`
       );
     }
-
-    if(!jsonld.hasValue(verificationMethod, 'type', this.requiredKeyType)) {
+    if(!jsonld.hasValue(verificationMethod, 'type', requiredKeyType)) {
       throw new Error(
-        `Invalid key type. Key type must be "${this.requiredKeyType}".`);
```

Comment:
I'm wondering what the situation around `this.requiredKeyType` is now.  Is it still a thing?

Reply:
Once I remove it from here, then it's not a thing anymore. So, I can remove it.

Reply:
Removed fce8215

Reply:
Made further comment on this elsewhere, marking this resolved.